src/unshadow.c

```diff
@@ -97,17 +97,36 @@ static void read_file(char *name, void (*process_line)(char *line))
 	if (fclose(file)) pexit("fclose");
 }
 
+static int spaceTab(char c) {
+	if (' ' == c) {
+		return 1;
+	}
+	if ('\t' == c) {
+		return 1;
+	}
+
+	return 0; // false
+}
+
 static void process_shadow_line(char *line)
 {
 	static struct shadow_entry **entry = NULL;
 	struct shadow_entry *last;
 	char *login, *passwd, *tail;
 
-	if (!(passwd = strchr(line, ':'))) {
-		while (*line == ' ' || *line == '\t') line++;
-		/* AIX */
-		if (!strncmp(line, "password = ", 11) && entry)
-			(*entry)->passwd = str_alloc_copy(line + 11);
+	if (!(passwd = strchr(line, ':'))) { /* AIX "password = " */
+		while (spaceTab(*line)) { line++; } /* skip space and tab */
```

Comment:
This also breaks our code style (although the original code did too). Should be:
```c
    /* AIX "password = " */
    if (!(passwd = strchr(line, ':'))) {
        /* skip space and tab */
        while (spaceTab(*line))
            line++;
```
More instances of same problem below.

Reply:
kr style? indent with space?

Reply:
Indent with tabs, align with spaces. Some minimum sanity is fixed with
`indent -kr -i8 -nlp -nbbo -ncs -l79 -lc79 -il0` plus, if supported, `-bad`

Reply:
I have fixed this in the updated commit.

@AlexanderLX42 Please test the latest commit to see that everything is working. I think `git reset --hard ...` and `git pull` commands will be required on your end to fetch the latest commits.

Reply:
@magnumripper I am thinking of renaming the `spaceTab` function name to `space_or_tab` to match the existing style in the file. Does this sound logical to you?

Reply:
Yeah that's also more matching our code style (which is still not really documented despite the GSoC work, or is it?)

Reply:
We could also use `isspace()` from ctype.h instead of rolling our own. But this works.